packages/fxa-content-server/app/tests/spec/lib/payment-server.js

```diff
@@ -19,7 +19,7 @@ describe('lib/payment-server-redirect', () => {
       subscriptions: {
         managementClientId: 'MOCK_CLIENT_ID',
         managementScopes: 'MOCK_SCOPES',
-        managementTokenTTL: 900,
+        managementTokenTTL: 1800,
```

Comment:
You probably don't *need* to update this in all the tests, since expiration is just a mock value and isn't *really* exercised time-wise.